normative_rule_defs/zihintntl.yaml

```diff
@@ -8,6 +8,7 @@ $schema: "../docs-resources/schemas/defs-schema.json#"
 
 chapter_name: Zihintntl (Non-tmp Hint)
 
+
```

Comment:
Should  this be a normative rule?

`When an NTL instruction is applied to a prefetch hint in the Zicbop extension, it indicates that a cache line should be prefetched into a cache that is outer from the level specified by the NTL.`